


export const getBusiness = async(req,res)=>{
    try {
        // const businesses = 
    } catch (error) {
        console.error("Hubo un error al conectarse a la persistencia de Business");
        res.status(500).send({error:error})
    }
}

export const getBusinessById = async(req,res)=>{
    
}

export const getBusinessByCategory = async(req,res)=>{
    
}

export const saveBusiness = async(req,res)=>{
    
}